simplq/src/components/common/Popup/TempQueueWarning.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Move to a separate folder named `ribbon/` inside `common/` with two files,
```
/common/Ribbon/
  - index.jsx                          // for the component
  - ribbon.module.scss                 // for styles
``` 

You can use [LoginButton](https://github.com/shubh128/simplQ-frontend/tree/master/simplq/src/components/common/LoginButton) for refernce.